src/components/event-form/event-form-controls.tsx

```diff
@@ -92,22 +100,41 @@ export const EventFormControls = ({
 				)}
 			/>
 			{withPartners && (
-				<Controller
-					control={control}
-					name="partners"
-					render={({ field: { onChange } }) => (
-						<Input
-							type="file"
-							accept="image/*"
-							multiple
-							label="Partners' logos"
-							isRequired
-							onChange={({ target: { files } }) =>
-								files && onChange([...files])
-							}
-						/>
-					)}
-				/>
+				<>
+					<Controller
+						control={control}
+						name="partners"
+						render={({ field: { onChange } }) => (
+							<Input
+								type="file"
+								accept="image/*"
+								multiple
+								label="Partners' logos"
+								isRequired
+								onChange={({ target: { files } }) =>
+									files && onChange([...files])
+								}
+							/>
+						)}
+					/>
+					<Controller
+						control={control}
+						name="partnerType"
+						render={({ field: { onChange } }) => (
+							<RadioGroup
+								isRequired
+								label="Select partner type"
+								orientation="horizontal"
+								defaultValue="event-partner"
+								onChange={onChange}
+							>
+								<Radio value="event-partner">Event partner</Radio>
+								<Radio value="sponsor">Sponsor</Radio>
+								<Radio value="only-logo">Only logo</Radio>
+							</RadioGroup>
+						)}
+					/>
```

Comment:
Ten komponent powinien być kontrolowany, takie jest przeznaczenie `Controller` (czyli przekazujemy `value` i `onChange` w propsie). Przydałoby się jeszcze ustawić `defaultValue`, ale wydaje mi się, że w `useForm` nie będzie się dało tego zrobić, więc przychodzi mi takie coś do głowy:
```suggestion
					<Controller
						control={control}
						name="partnerType"
						render={({ field: { value = 'event-partner', ...field } }) => (
							<RadioGroup
								isRequired
								label="Select partner type"
								orientation="horizontal"
								value={value}
								{...field}
							>
								<Radio value="event-partner">Event partner</Radio>
								<Radio value="sponsor">Sponsor</Radio>
								<Radio value="only-logo">Only logo</Radio>
							</RadioGroup>
						)}
					/>
```